### PR TITLE
Add missing warning for general find command in User Guide

### PR DESCRIPTION
**Pull request by FH-30**

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -273,6 +273,11 @@ To find flashcards, you can find them by specifying keywords within their questi
 
 :information_source: You must provide at least one search parameter.
 
+<div markdown="block" class="alert alert-danger">
+:exclamation: Take note that only flashcards which fullfill **all** the specified search parameters will be displayed. Flashcards that fullfill only some search parameters will not be diplayed.
```
Comment (josiahkhoo) [inline docs/UserGuide.md]: think there is a small typo here, should `fulfill` instead of `fullfill`

**Review by josiahkhoo — APPROVED** [review]
Looks good, just need to amend a typo!

File: docs/UserGuide.md (hunk 1 above)
Comment (FH-30) [reply docs/UserGuide.md]: Okay fixed it!

**Review by joshtyf — APPROVED** [review]
LGTM!

**Pull request merged**